package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@esm-bundle/angular",
-  "version": "17.3.4",
+  "version": "18.0.6",
```

Don't manually update the version:

```suggestion
  "version": "17.3.4",
```

---

updated, thank you for checking it!